### DISCUSSION TITLE
Configurable hotkeys

### DISCUSSION THREAD

**Discussion by asisix** [QL-Win/QuickLook#1215] [Ideas]
For now the Spacebar is used to show/hide the preview window. I would like to configure the mouse button for this. So that when I hold down the mouse button and the preview window appears and when I release it, the window closes. But at the same time leave a spacebar for a regular preview window (which does not close by itself)

**Comment by rabelux** [comment]
The hotkey cannot be changed currently but hold-to-preview has recently been added in c107732.

**Comment by xupefei** [comment]
Pretty sure you can map the mouse button to a keyboard key press using some software.
It will work as you wish since we have the hold-to-preview feature as rabelux mentioned.
